packages/web-sdk/src/instrumentations/webVitals/webVitalsWithAttribution.test.ts

```diff
@@ -74,7 +64,6 @@ jest.mock('web-vitals/attribution', () => {
     onINP: (cb: (metric: MetricWithAttribution) => void) => {
       cb(
         createMetric('INP', {
-          eventTime: 0.1,
```

Comment:
out of curiosity, why is this change needed?

Reply:
`INPAttribution` does not contain `eventTime` as a property anymore :-/.

<img width="722" alt="image" src="https://github.com/user-attachments/assets/6c2dbf92-80d0-4890-b3fb-9182491c8a1c" />